Libraries/LibJS/Runtime/Object.cpp

```diff
@@ -272,6 +302,12 @@ Value Object::get_by_index(i32 property_index) const
 
     const Object* object = this;
     while (object) {
+        if (is_string_object()) {
+            auto& string = static_cast<const StringObject*>(this)->primitive_string().string();
+            if (property_index < (i32)string.length())
+                return js_string(heap(), string.substring(property_index, 1));
+            return js_undefined();
+        }
```

Comment:
Suddenly implementing indexed access to `StringObject` in the middle of an unrelated patch huh? ;)

Reply:
yeah, this belongs in a separate commit. very desperately wanted to get all the tests (I partially wrote beforehand) to pass :smile: